GNUmakefile

```diff
@@ -29,7 +29,7 @@ USE_ASCENT_INSITU = FALSE
 
 WarpxBinDir = Bin
 
-USE_PSATD = FALSE
+USE_PSATD = TRUE
```

Comment:
Could you revert this default parameter to `FALSE`?

Reply:
Oh yes! Thank you for catching it.